sdk/signature-mpc-wasm/src/lib.rs

```diff
@@ -248,6 +248,14 @@ pub fn generate_keypair() -> Result<JsValue, JsErr> {
     Ok(serde_wasm_bindgen::to_value(&(public_key, private_key))?)
 }
 
+#[wasm_bindgen]
+pub fn generate_keypair_from_bytes(bytes: &[u8]) -> Result<JsValue, JsErr> {
```

Comment:
```suggestion
pub fn generate_keypair_from_keypair(keypair: &Ed25519Keypair) -> Result<JsValue, JsErr> {
```

Reply:
This won't work as `Ed25519Keypair` is not serializable to rust. I'll create a TS wrapper function.